scrapy/utils/project.py

```diff
@@ -70,6 +70,7 @@ def get_project_settings():
     # XXX: remove this hack
     pickled_settings = os.environ.get("SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE")
     if pickled_settings:
+        warnings.warn("Use of environmental variable 'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' is deprecated",DeprecationWarning)
```

Comment:
A few notes:
* We use `ScrapyDeprecationWarning` (from `scrapy.exceptions`) so that deprecation warnings are not silenced by default in Python 2.
* environmental variable → environment variable
* Please, split the warning about multiple lines. See [Coding Style](https://docs.scrapy.org/en/latest/contributing.html#coding-style).

Reply:
@Gallaecio Is it ok now? Please review.

Reply:
Not exactly. Breaking lines like that creates extra space between the words before and after line breaks. Please, take https://github.com/scrapy/scrapy/blob/master/scrapy/utils/log.py#L196-L198 as an example.

Reply:
@Gallaecio  I checked  whether breaking lines like that creates extra spaces, it didn't actually.Anyways I have made the changes according to the example.

```
>>> warnings.warn("Use of environment variable \
... 'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' \
... is deprecated",DeprecationWarning)
__main__:3: DeprecationWarning: Use of environment variable 'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' is deprecated

>>> warnings.warn("Use of environment variable "
... "'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE'"
... " is deprecated",DeprecationWarning)
__main__:3: DeprecationWarning: Use of environment variable 'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' is deprecated
```
Is that problem maybe specific to ScrapyDeprecationWarning?

Reply:
I would say it’s a problem specific to Python :)

```
[adrian@pondal ~]$ python3
Python 3.6.7 (default, Nov 27 2018, 21:03:46) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print("adsf \
...        asdf")
adsf        asdf
>>> 
```

In your test you are not using indentation, as in our code. The problem with using a backslash to escape a line break is that all the indentation spaces of the next line remain.